Bottom: 87d49d35849a2413bd1740722b52c372d76e3835
Top:    7ca527dff8040c8165692dedbb8bce3b168f3d7c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 04:25:16 +0100

Refresh of fix-bug-with-polygon-boolean-o

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 4583732..336dace 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1593,7 +1593,7 @@ cntr_Collect (jmp_buf * e, PLINE ** A, POLYAREA ** contours, PLINE ** holes,
 	      PutContour (e, tmprev, contours, holes, owner, NULL, NULL);
 	      return TRUE;
 	    }
-          /* Fall through (I think this is correct. pcjc2) */
+	  /* break; */ /* Fall through (I think this is correct! pcjc2) */
 	case PBO_UNITE:
 	case PBO_SUB:
 	  if ((*A)->Flags.status == OUTSIDE)
